ext/test/w3c_tracecontext_test/main.cc

```diff
@@ -89,11 +89,13 @@ struct Uri
 class NoopEventHandler : public http_client::EventHandler
 {
 public:
-  void OnEvent(http_client::SessionState state, nostd::string_view reason) noexcept override {}
+  void OnEvent(http_client::SessionState /* state */,
```

Comment:
Is this introduced to workaround some compilation warning? Wrap the parameter name in inline comment seems more verbose than the original parameter there.

Reply:
Yes, clang will trigger `-Werror,-Wunused-parameter` when compiling with `-Wextra -Werror`.